include/arch/arm/arch/machine/debug.h

```diff
@@ -248,21 +327,21 @@ static inline syscall_error_t Arch_decodeUnsetBreakpoint(tcb_t *t, uint16_t bp_n
         return ret;
     }
 
+#ifdef CONFIG_ARCH_AARCH32
```

Comment:
This ifdef can go now that you have the aarch64 `Arch_breakpointIsSingleStepping()` returning false.